app/controllers/concerns/hydra/collections_controller_behavior.rb

```diff
@@ -175,7 +175,7 @@ def move_members_between_collections
     # include filters into the query to only include the collection memebers
     def include_collection_ids(solr_parameters, user_parameters)
       solr_parameters[:fq] ||= []
-      solr_parameters[:fq] << Solrizer.solr_name(:collection, :facetable)+':"'+@collection.id+'"'
+      solr_parameters[:fq] << "{!join from=hasCollectionMember_ssim to=id}id:#{@collection.id}"
```

Comment:
@jcoyne this is fantastic!  Is this a solr join?  What version of solr is required for this to work?  The README may need to be updated to reflect the new dependancy. Or has it been available all along.